.github/workflows/master.yml

```diff
@@ -52,27 +52,33 @@ jobs:
     - name: Install system dependencies
       run: |
         sudo apt-get install -y libcurl4-openssl-dev
-    - name: Install dependencies
+    - name: Install R dependencies
+      shell: Rscript {0}
       run: |
         install.packages("devtools")
         remotes::install_deps('mlflow/R/mlflow', dependencies = TRUE, upgrade = FALSE)
-      shell: Rscript {0}
-    - name: Run tests
+    - name: Build package
+      working-directory: mlflow/R/mlflow
       run: |
-        source ./dev/install-common-deps.sh
```

Comment:
Removed `./dev/install-common-deps.sh` because the test environment is created in `.create-test-env.R`